src/main/java/com/wanted/codebombalms/lecture/README.md

```
@@ -69,3 +69,10 @@ lecture
 | `course` | 강의가 속할 강좌의 존재 여부와 상태 확인 |
 | `learning` | 강의 진행률 기록과 조회의 기준 |
 | `problems` | 강의에 연결된 문제 세트 조회 기준 |
+# Lecture Problem Set API
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)